packages/ui/src/components/vuestic-components/va-switch/VaSwitch.spec.js

```diff
@@ -0,0 +1,26 @@
+import Vue from 'vue'
+import { shallowMount } from '@vue/test-utils'
+import VaSwitch from './VaSwitch'
+
+import { ColorThemePlugin } from '../../../services/ColorThemePlugin'
```

Comment:
Component should work without color theme plugin. Ideally we want to test without it.